app/src/main/cpp/skyline/jvm.cpp

```diff
@@ -104,7 +109,41 @@ namespace skyline {
         env->CallVoidMethod(instance, clearVibrationDeviceId, index);
     }
 
+    jobject JvmManager::ShowKeyboard(applet::swkbd::KeyboardConfigVB &config, std::u16string initial_text) {
+        auto buff{env->NewDirectByteBuffer(&config, sizeof(applet::swkbd::KeyboardConfigVB))};
+        auto str{env->NewString(reinterpret_cast<const jchar *>(initial_text.data()), static_cast<int>(initial_text.length()))};
+        jobject localKeyboardDialog = env->CallObjectMethod(instance, showKeyboardId, buff, str);
+        env->DeleteLocalRef(buff);
+        env->DeleteLocalRef(str);
+        auto keyboardDialog = env->NewGlobalRef(localKeyboardDialog);
+        env->DeleteLocalRef(localKeyboardDialog);
+        return keyboardDialog;
```

Comment:
A newline before this would be appreciated as it separates it into two logically isolated parts.